portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includePageNav.vm

```diff
@@ -161,7 +161,7 @@
 				#end ## END of IF (${sitePages.pageNavShowHelp})
 	
 				<li id="toolsNav-toggle-li" class="Mrphs-toolsNav__menuitem Mrphs-collapseTools #if ($toolsCollapsed) min #else max #end">
-					<button type="button" aria-pressed="$toolsCollapsed" title="${rloader.sit_toggle_nav}" class="Mrphs-toggleNav js-toggle-nav btn-link  #if ($toolsCollapsed) min #else max #end">
+					<button type="button" aria-pressed="#if(${toolsCollapsed})true#else#**#false#end" title="${rloader.sit_toggle_nav}" class="Mrphs-toggleNav js-toggle-nav btn-link  #if ($toolsCollapsed) min #else max #end">
```

Comment:
Why the #**# ?

Reply:
cool trick right? 

the idea is to avoid any space

#elsefalse (does not work)

so need to insert a comment there